wikitemplate-IPFS.md

```diff
@@ -210,6 +243,8 @@
 - [ ] Verify the setting is enabled by default.
 - [ ] Verify disable/enable setting is retained between browser launch/restarts.
 - [ ] Verify setting state is retained during upgrade.
+- [ ] Verify you can't change the IPFS public gateway address to `https://cloudflare-ipfs.com/` (An error informing that only gateways with proper Origin isolation can be used in browser context is displayed)
+- [ ] Verify you you can change the IPFS public gateway address to `https://cf-ipfs.com/`  (provides origin isolation)
```

Comment:
Requires https://github.com/brave/brave-browser/issues/16999

Reply:
nit: repeating you

Reply:
Nit: I"m pedantic; can you include periods, just for consistency's sake, while you're in there?

Reply:
you've got both `Origin isolation` and `origin isolation` here , just FYI